docs(api):docs-pipette-flow-rates-mismatch

jwwojak opened the pull request:
# Overview

In [Pipette Flow Rates](https://docs.opentrons.com/v2/pipettes/characteristics.html#pipette-flow-rates) there's a code block that shows default flow rate for the 1000 µL pipette as 160 µL/s. Later on that same page the flow rate table lists the 1000 µL pipette flow rate as 716 µL, which is correct.

The fix here: Make sure the default flow rate listed in the code block on lines 161, 162 match the default flow rate published in a table later on the same page. Add info about 50 µL pipettes using 50 µL tips only.

Sandbox:

- [Pipette Flow Rates](http://sandbox.docs.opentrons.com/docs-pipette-flow-rates-mismatch/v2/pipettes/characteristics.html#pipette-flow-rates)
- [Flex Pipette Flow Rates](http://sandbox.docs.opentrons.com/docs-pipette-flow-rates-mismatch/v2/pipettes/characteristics.html#flex-pipette-flow-rates)

 Jira:  RTC-421

## Test Plan and Hands on Testing

Have another trusted source review the change.

## Changelog

- Code block at around lines 161, 162 show incorrect flow rate.
- Change code block sample code to 716 µL/s from 160 µL/s.
- Update flow rate table to indicate a 50 µL pipette should only use 50 µL tips.

## Review requests

Seem like 2 simple changes. Are these what you anticipated? 

## Risk assessment

Low. Docs changes only.

jwwojak committed:
Code block and flow rate mismatch

Make sure the default flow rate listed in the code block on lines 161, 162 match the default flow rate published in a table later on the same page. Add info about 50 ul pipettes using 50 ul tips only.

ecormany commented on api/docs/v2/pipettes/characteristics.rst:
```diff
@@ -158,8 +158,8 @@ These flow rate properties operate independently. This means you can specify dif
-        pipette.dispense(200, plate["A2"])  # 160 µL/s
+        pipette.aspirate(200, plate["A1"])  # 716 µL/s
+        pipette.dispense(200, plate["A2"])  # 716 µL/s
         pipette.blow_out()                  #  80 µL/s
```
The app reports default blowout as having the same speed as default aspirate/dispense.
<img width="597" alt="image" src="https://github.com/user-attachments/assets/f51d6d19-df25-4953-b4c6-35bdeb9a84aa">
This intuitively seems right? "blowing out" at a tenth the dispense speed seems way too weak.

ecormany replied:
Yep, confirmed this in the pipette definition. Let's update it. 
https://github.com/Opentrons/opentrons/blob/f7122e580a9ff741d18a6d467a255505229e1d03/shared-data/pipette/definitions/2/liquid/single_channel/p1000/default/3_7.json#L83-L86

jwwojak replied:
New push to correct default blow out rate on line 163 (should be 716 ul/s). Also, update following code example, line 172 to have more realistic custom blow out rate.

jwwojak committed:
Fix blow_out rates too

Ed noticed that the blow out rate of 70-ish ul/s seemed low. Checking, the default for this pipette typs is 716 ul/s. Updating code examples to match correct rate. Fix following example to be more accurate w/ custom flow rate.

ecormany reviewed (approved):
🚢💨
